### PR TITLE
Fix log message

### PR DESCRIPTION
**Pull request by chilagrow**
# Description

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/clientconn/listener.go [hunk 1]
```diff
@@ -111,22 +113,22 @@ func Listen(opts *NewListenerOpts) (*Listener, error) {
...
-		ll.Info(fmt.Sprintf("Listening on Unix %s...", l.UnixAddr()))
+		ll.InfoContext(ctx, fmt.Sprintf("Listening on Unix %s...", l.UnixAddr()))
 	}
 
 	if l.TLS != "" {
 		var config *tls.Config
 
 		if config, err = tlsutil.Config(l.TLSCertFile, l.TLSKeyFile, l.TLSCAFile); err != nil {
-			return nil, err
+			return nil, lazyerrors.Error(err)
```
Comment (AlekSi) [inline internal/clientconn/listener.go:123]: That's one of the few places where the error string is user-visible. Try to start FerretDB with a non-existing TLS file and see the output. It should be nice, and lazyerrors will make it worse.

Let's add a comment so we don't forget.

**Review by rumyantseva — COMMENTED** [review]
Looks good apart from what Alexey noted.

I see we use `...` almost everywhere, but not on the line `211`. Personally, I like having `...` because they give an idea that the process hasn't finished yet (but it's not a strong opinion).

File: internal/clientconn/listener.go (hunk 1 above)
Comment (chilagrow) [reply internal/clientconn/listener.go:123]: Indeed, lazyerror is not pretty. Added a comment

**Comment by chilagrow** [comment]
> Looks good apart from what Alexey noted.
> 
> I see we use `...` almost everywhere, but not on the line `211`. Personally, I like having `...` because they give an idea that the process hasn't finished yet (but it's not a strong opinion).

I've left `...` in L211

File: internal/clientconn/conn.go [hunk 2]
```diff
@@ -167,7 +167,7 @@ func (c *conn) run(ctx context.Context) (err error) {
 
 	defer func() {
 		if p := recover(); p != nil {
-			c.l.LogAttrs(ctx, logging.LevelDPanic, fmt.Sprint("Failed to run", p), logging.Error(err))
+			c.l.LogAttrs(ctx, logging.LevelDPanic, fmt.Sprint("Failed to run: ", p), logging.Error(err))
```
Comment (AlekSi) [inline internal/clientconn/conn.go:170]: If that PR intends to sync changes with v2 (as the branch name indicates), then this line looks different there

**Review by AlekSi — COMMENTED** [review]
Let's that be the last PR that fixes logging for a while because I'm concerned that we are in the endless loop of "port those changes with slight improvements that we need to port back (with slight improvements…)"

File: internal/clientconn/conn.go (hunk 2 above)
Comment (chilagrow) [reply internal/clientconn/conn.go]: Yeah let's make the same.

**Review by rumyantseva — APPROVED** [review]
Looks good!

**Review by noisersup — APPROVED** [review]

**Pull request merged**